### DISCUSSION TITLE
v1.4.1 AUX Audio Update

### DISCUSSION THREAD

**Discussion by MultimediaLucario** [MultimediaLucario/Lenovo-ThinkPad-T480#5] [Announcements]
![repository-t480-template](https://github.com/MultimediaLucario/Lenovo-ThinkPad-T480/assets/72415505/5ee730f6-6717-4bff-a9e8-e8a1d95b7a7d)

# Audio Bug has been patched. 🎧 

This release includes a patch for a small bug for when any wired AUX devices are connected and 3 other EFI folders. All of which are pretty self-explanatory and are equipped with the ```MacBookPro15,2``` or 2018 MacBook Pro SMBIOS.

**Full Changelog**: https://github.com/MultimediaLucario/Lenovo-ThinkPad-T480/compare/v1.4.0...v1.4.1

<hr /><em>This discussion was created from the release <a href='https://github.com/MultimediaLucario/Lenovo-ThinkPad-T480/releases/tag/v1.4.1'>v1.4.1 AUX Audio Update</a>.</em>

**Comment by MalikBehloul** [comment]
Hello Bluetooth is not stable !!!

**Reply by MultimediaLucario** [reply]
Have you tried rebooting your laptop?

**Comment by MalikBehloul** [comment]
Yes but when back from sleep it no longer works I am on Intel 8265/ 8275

**Reply by MultimediaLucario** [reply]
I just dropped a new release, please let me know if this fixed your issue.
